pages/common/sftp.md

```diff
@@ -11,6 +11,10 @@
 
 `sftp -P {{remote_port}} {{remote_host}}`
 
+- Navigate through the LOCAL system file hierarchy:
+
+`lpwd, lls, lcd`
```

Comment:
While I am totally for including those three commands, it should probably rather be split in three separate commands with an explanation each